diagnostic.md

```diff
@@ -58,18 +58,18 @@ What does typing `bin/rails dbconsole` in a directory that houses a rails app do
 us?
 
 ```md
-// your response here
+This will start rails in the terminal.
 ```
 
 What does typing `rails db` in a directory that houses a rails app do for us?
 
 ```md
-// your response here
+This will give list a variety of options regarding a potential rails app
```

Comment:
`rails db` opens up a PostgreSQL shell connected to the database that we're working with.